Library/Homebrew/brew.sh

```diff
@@ -38,11 +38,16 @@ then
   export LC_ALL="en_US.UTF-8"
 fi
 
-# Where we store built products; /usr/local/Cellar if it exists,
-# otherwise a Cellar relative to the Repository.
-if [[ -d "$HOMEBREW_PREFIX/Cellar" ]]
+# Where we store built products; Cellar under the prefix if it exists or the
+# prefix is /usr/local (so people can get as many bottles as possible)
+# otherwise a Cellar relative to the repository.
+if ! [[ -d "$HOMEBREW_PREFIX/Cellar" ]]
 then
   HOMEBREW_CELLAR="$HOMEBREW_PREFIX/Cellar"
+elif [[ "$HOMEBREW_PREFIX" == "/usr/local" &&
```

Comment:
Please use a single `=` for string comparison in shell code for consistency with existing code.